Docs/Debugging/offline.md

```diff
@@ -1,317 +0,0 @@
-# Offline Development and Debugging with mbed
-
-At the heart of mbed we have an online compiler. While that is incredibly convenient for getting a project started or while prototyping, chances are that at some point you'll miss a debugger, or you'll want to develop while not having an active internet connection. Fortunately, we support exporting applications to a number of toolchains - including GCC, uVision and Eclipse - and committing source code back to the online environment. Combining the best of both the online and offline world.
-
-In this article we'll:
-
-* Cover ways to set up your local toolchain.
-* Show how to debug applications with your favourite IDE.
-* Explain how to sync online and offline code and libraries.
```

Comment:
1. Should be covered by mbed CLI and by toolchain.md in this repo
2. Is covered by this repo
3. Should be covered by mbed CLI